packages/mui-base/src/TrapFocus/TrapFocus.test.js

```diff
@@ -283,6 +283,28 @@ describe('<TrapFocus />', () => {
     expect(screen.getByTestId('root')).toHaveFocus();
   });
 
+  it('does not allow sentinelStart or sentinelEnd to be tabbable until open={true}', () => {
```

Comment:
Instead of testing the `tabindex` value, I would suggest testing the behavior, as the implementation may change in the future.

Reply:
@mnajdova done! However, to add _real_ tabbability testing, I needed to add [`@testing-library/user-event`](https://www.npmjs.com/package/@testing-library/user-event) which has an awkward naming conflict with `test/utils/userEvent.ts` and I don't know if you have a preferred solution for that situation.